### PR TITLE
chore: rename hlx3 to hlx

### PR DESCRIPTION
**Pull request by tripodsan**
Test URL: https://hlx3-hlx--helix-project-boilerplate--adobe.hlx3.page/?rum=on

### CODE REVIEW

File: scripts/scripts.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ export function sampleRUM(checkpoint, data = {}) {
       const sendPing = () => {
         // eslint-disable-next-line object-curly-newline, max-len, no-use-before-define
         const body = JSON.stringify({ weight, id, referer: window.location.href, generation: RUM_GENERATION, checkpoint, ...data });
-        const url = `https://rum.hlx3.page/.rum/${weight}`;
+        const url = `https://rum.hlx.page/.rum/${weight}`;
```
Comment (trieloff) [inline scripts/scripts.js:38]: Not sure about that. Let me test.

Comment (trieloff) [reply scripts/scripts.js:38]: Now it works.

**Review by trieloff — APPROVED** [review]

File: scripts/scripts.js (hunk 1 above)
Comment (trieloff) [reply scripts/scripts.js:38]: @stefan-guggisberg fyi, added the domain in https://manage.fastly.com/compute/services/5Qir8H8bLeaRuldIp9TWq4/versions/39/domains

File: scripts/scripts.js [hunk 2]
```diff
@@ -404,7 +404,7 @@ export function normalizeHeadings(el, allowedHeadings) {
 export function makeLinksRelative(main) {
   main.querySelectorAll('a').forEach((a) => {
     // eslint-disable-next-line no-use-before-define
-    const hosts = ['hlx3.page', 'hlx.page', 'hlx.live', ...PRODUCTION_DOMAINS];
+    const hosts = ['hlx.page', 'hlx.page', 'hlx.live', ...PRODUCTION_DOMAINS];
```
Comment (rofe) [inline scripts/scripts.js:407]:
```suggestion
    const hosts = ['hlx.page', 'hlx.live', ...PRODUCTION_DOMAINS];
```

**Review by rofe — CHANGES_REQUESTED** [review]

**Review by rofe — APPROVED** [review]

**Pull request merged**